server/src/main/java/org/elasticsearch/indices/analysis/AnalysisModule.java

```diff
@@ -153,7 +150,6 @@ public AnalysisRegistry getAnalysisRegistry() {
 
         // Add filters available in lucene-core
         preConfiguredTokenFilters.register("lowercase", PreConfiguredTokenFilter.singleton("lowercase", true, LowerCaseFilter::new));
-        preConfiguredTokenFilters.register("standard", PreConfiguredTokenFilter.singleton("standard", false, StandardFilter::new));
```

Comment:
same here

Reply:
This is https://github.com/elastic/elasticsearch/pull/33310/commits/67efaae12d76f658530555006a4388df863002a4